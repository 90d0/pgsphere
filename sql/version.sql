-- Check current pgsphere version
SELECT pg_sphere_version();

