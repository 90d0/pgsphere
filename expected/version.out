-- Check current pgsphere version
SELECT pg_sphere_version();
 pg_sphere_version 
-------------------
 1.4.0
(1 row)

